fxa-oauth-server/lib/routes/verify.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
   validate: {
     payload: {
       token: validators.token.required(),
-      email: Joi.boolean().optional()
+      email: Joi.boolean().optional(),
+      type: Joi.string().valid('access_token', 'refresh_token').optional(),
```

Comment:
This endpoint is already very close to the "token introspection endpoint" from [RFC7662](https://tools.ietf.org/html/rfc7662), and that spec has provisions for checking refresh tokens as well as access tokens.  We could use this as an opportunity to align with prior art.